File: vetor/exercicio7.py
import math

numeros = [0] * 100

for i in range(100):
    numeros[i] = int(input(f"Digite o número {i+1}: "))

soma_quadrados = 0
for i in range(100):
    soma_quadrados += numeros[i] ** 2

raiz_media_quadratica = math.sqrt(soma_quadrados / 100)

print(f"A raiz média quadrática do conjunto é: {raiz_media_quadratica:.2f}")

File: vetor/exercicio7.c
#include <stdio.h>
#include <math.h> 

int main() {
    int vetor[100];
    double somaQuadrados = 0.0;
    double rms;

    printf("Digite 100 números inteiros:\n");
    for (int i = 0; i < 100; i++) {
        printf("Número %d: ", i + 1);
        scanf("%d", &vetor[i]);
        somaQuadrados += vetor[i] * vetor[i];
    }

    rms = sqrt(somaQuadrados / 100);

    printf("\nRaiz Média Quadrática (RMS): %.2f\n", rms);

    return 0;
}

